src/seedu/addressbook/ui/TextUi.java

```diff
@@ -168,5 +169,14 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    //Returns a concatenated version of the printable strings of each object.
+    public String getPrintableString(Printable... printables){
+        String output = null; 
+        for (Printable p : printables) {
+            output += p.getPrintableString() + ", ";
```

Comment:
Doesn't this result in an extra `,` at the end? You could have used `StringJoiner` or `String.join` instead.